kibitzr/transformer/xpath.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import re
+
+import six
+from defusedxml import lxml as dlxml
+from lxml import etree
```

Comment:
lxml is a relatively big library and sometimes hard to install.
Old code imported `lxml` (as well as `re` and `defusedxml`) from within function to save some memory. And to be able to work without lxml installed (if it's not needed).

Reply:
Makes sense. Moved imports inside functions.
But shouldn't lxml and defusedxml be moved to extras_require in setup.py?